frontend/app/layout.js

```diff
@@ -1,34 +1,29 @@
-import localFont from "next/font/local";
-import "./_styles/globals.css";
-import { SessionProvider } from "next-auth/react";
+import localFont from 'next/font/local';
+import './_styles/globals.css';
 
 const geistSans = localFont({
-  src: "./fonts/GeistVF.woff",
-  variable: "--font-geist-sans",
-  weight: "100 900",
+  src: './fonts/GeistVF.woff',
```

Comment:
you folks reaaaaaally need a eslint in your project config that is shared in repo, this is one of many times I see change of quotes in the diff and this is due to the fact that you don't have streamlined config as the team! Please look into this, this is part of good practice :)